.gitattributes

```diff
@@ -0,0 +1 @@
+*.x text -merge eol=lf
```

Comment:
What does the `-merge` do?

Reply:
Ah it turns off 3-way merge? (Assuming if I'm understanding [this](https://git-scm.com/docs/gitattributes#_merge) correctly.) That seems unrelated to the rest of this change. Why is it needed?